ldk/javascript/src/document/index.ts

```diff
@@ -41,3 +48,47 @@ export function xlsxDecode(data: Uint8Array): Promise<Workbook> {
 export function readPDF(data: Uint8Array): Promise<PDFOutput> {
   return promisifyWithParam(mapper.mapToBinaryData(data), oliveHelps.document.readPDF);
 }
+
+export function readPDFWithOcr(data: Uint8Array): Promise<PDFOutputWithOcrResult> {
+  return new Promise((resolve, reject) => {
+    try {
+      let pdfOutputResult = {};
+      const ocrResults: OCRResults = {};
+      oliveHelps.document.readPDF(mapper.mapToBinaryData(data), (error, pdfOutput) => {
+        if (error) {
+          console.error(`Received error on result: ${error.message}`);
+          reject(error);
+          return;
+        }
+        console.log(pdfOutput);
+        pdfOutputResult = pdfOutput;
+        if (pdfOutput != null) {
+          Object.entries(pdfOutput).forEach(([page, { content }]) => {
+            content.forEach((item) => {
+              if (item.type === 'photo') {
```

Comment:
the only thing I am concerning about is the ocr result may not align the order of text exactly same as the image. 
Do you think if it is better to still give user access to the image from PDF? Maybe we could add a new type to return text for image.

Reply:
yeah i went back on forth just because the string of photo data can be pretty big but its probably better to include it. I do like the idea of a new type representing text from a photo